packages/lesswrong/components/posts/PostsItemTitle.jsx

```diff
@@ -3,12 +3,14 @@ import React from 'react';
 import Typography from '@material-ui/core/Typography';
 import { withStyles } from '@material-ui/core/styles'
 import grey from '@material-ui/core/colors/grey';
+import classNames from 'classnames';
 
 const styles = theme => ({
   root: {
     whiteSpace:"nowrap",
     overflow:"hidden",
     textOverflow:"ellipsis",
+    textShadow: "0px 0px 0px rgba(0,0,0,.87)",
```

Comment:
What's this for? The zero-pixel radius theoretically would make it a no-op, except that antialiasing makes it not. It seems like it should make the text slightly bolder, but I'd worry about its appearance varying a lot between operating systems. If we can do this with `font-weight` instead, that would be better.

Reply:
This is specifically the thing Said recommends doing since fontWeight is surprisingly terrible. It makes it very slightly bolder but not much.

http://localhost:3000/posts/PfgHiMZ3jF2fifssH/changing-main-content-font-to-valkyrie#aEuSFxadATbc4FQJJ

Reply:
```suggestion
    textShadow: "0px 0px 0px rgba(0,0,0,.87)", // Make the text bolder (discussion: http://lesswrong.com/posts/PfgHiMZ3jF2fifssH/changing-main-content-font-to-valkyrie#aEuSFxadATbc4FQJJ)
```